src/formatDate.js

```diff
@@ -8,7 +8,31 @@
  * @returns {string}
  */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const splittedDate = date.split(fromFormat[fromFormat.length - 1]);
+  const dateProperties = {};
+  const updatedDate = [];
+
+  for (let i = 0; i < 3; i++) {
+    dateProperties[fromFormat[i]] = splittedDate[i];
+  }
+
+  if (fromFormat.includes('YYYY') && toFormat.includes('YY')) {
+    dateProperties['YY'] = dateProperties['YYYY'].slice(-2);
```

Comment:
Good job on handling the year conversion from `YYYY` to `YY`. The code is concise and understandable.